### PR TITLE
Updating headers to be string type, conversion handled on worker

### LINKED ISSUE #78: Non-string header values on http response throws error
See details on the original issue reported here: https://github.com/Azure/azure-functions-host/issues/2768
Initial fix was to update RpcHttp proto file to <string, TypedData> dictionary. According to [HttpSpec](https://tools.ietf.org/html/rfc2616#section-4.2) header values are only valid ASCII characters. Need to fix NodeJS worker to convert to strings before sending message to functions host

### PR DESCRIPTION
**Pull request by mhoeger**
Resolves #78

### CODE REVIEW

File: src/Converters.ts [hunk 1]
```diff
@@ -34,9 +26,9 @@ export function toRpcHttp(inputMessage): rpc.TypedData$Properties {
 }
 
 export function toRpcHttpHeaders(inputHeaders: rpc.TypedData$Properties) {
-  let rpcHttpHeaders: {[key: string]: rpc.TypedData$Properties} = {};
+  let rpcHttpHeaders: {[key: string]: string} = {};
   for (let key in inputHeaders) {
-    rpcHttpHeaders[key] = toTypedData(inputHeaders[key]);
+    rpcHttpHeaders[key] = inputHeaders[key].toString();
```
Comment (pragnagopa) [inline src/Converters.ts]: Check for null or empty values inputHeaders[key]?

File: src/Converters.ts [hunk 2]
```diff
@@ -34,9 +26,11 @@ export function toRpcHttp(inputMessage): rpc.TypedData$Properties {
 }
 
 export function toRpcHttpHeaders(inputHeaders: rpc.TypedData$Properties) {
-  let rpcHttpHeaders: {[key: string]: rpc.TypedData$Properties} = {};
+  let rpcHttpHeaders: {[key: string]: string} = {};
   for (let key in inputHeaders) {
-    rpcHttpHeaders[key] = toTypedData(inputHeaders[key]);
+    if (inputHeaders[key] != null) {
+      rpcHttpHeaders[key] = inputHeaders[key].toString();
```
Comment (pragnagopa) [inline src/Converters.ts]: Add test setting the response headers of different types "stringHeader" , 123

File: src/Converters.ts [hunk 3]
```diff
@@ -34,9 +26,11 @@ export function toRpcHttp(inputMessage): rpc.TypedData$Properties {
 }
 
 export function toRpcHttpHeaders(inputHeaders: rpc.TypedData$Properties) {
-  let rpcHttpHeaders: {[key: string]: rpc.TypedData$Properties} = {};
+  let rpcHttpHeaders: {[key: string]: string} = {};
   for (let key in inputHeaders) {
-    rpcHttpHeaders[key] = toTypedData(inputHeaders[key]);
+    if (inputHeaders[key] != null) {
```
Comment (pragnagopa) [inline src/Converters.ts]: if(inputHeaders[key]) should take care of undefined , null

Comment (mhoeger) [reply src/Converters.ts]: it evaluates to false with a value of 0 though :(

Comment (pragnagopa) [reply src/Converters.ts]: Ahh. Thanks for checking. Add a test for 0 too.

**Pull request merged**